src/Components/OATPropertyEditor/Internal/PropertyList/PropertyList.tsx

```diff
@@ -182,9 +211,24 @@ const PropertyList: React.FC<IPropertyListProps> = (props) => {
                     property.name
                 );
                 updatedProperties[index].name = args.name;
-                updateReference({
+                const updatedReference = {
                     ...selectedItem,
                     properties: updatedProperties
+                };
+                // validate changes
+                validateOntology({
+                    existingModels: oatPageState.currentOntologyModels,
+                    selectedModelId: oatPageState.selection?.modelId,
+                    originalItem: selectedItem,
+                    updatedItem: updatedReference
+                }).then((validationResult) => {
```

Comment:
came here, should you also have `.catch` block for this promise to call `showErrorDialog(error);` in case validateOntology fails?

Reply:
done